tokio/Cargo.toml

```diff
@@ -51,7 +50,8 @@ full = [
 blocking = ["rt-core"]
 dns = ["rt-core"]
 fs = ["rt-core", "io-util"]
-io-driver = ["mio", "lazy_static"]
+io-driver = ["mio", "lazy_static"] # internal only
+io-readiness = [] # internal only
```

Comment:
It can feel a little weird... but if we point out they are internal-only features, it's not part of the public API. The alternative is to have `#[cfg]` checking for tcp, udp, process, signal, uds, etc. That felt more fragile, but maybe it's better. I don't know.

Reply:
My gut is to agree w/ @hawkw. Up until now, we've dealt w/ "internal" dependencies using the `cfg_` macros. Is there a reason why those don't work? Is it a question of duplicating the mio and lazy_static dependencies?